pele_platform/Adaptive/simulation.py

```diff
@@ -19,10 +19,11 @@
 import pele_platform.Utilities.Helpers.Metals.metal_polarisation as mp
 import pele_platform.Adaptive.metrics as mt
 import pele_platform.Utilities.Helpers.water as wt
-import pele_platform.Adaptive.ligand_parametrization as lg
+from pele_platform.Adaptive import Parametrizer
 import pele_platform.Adaptive.box as bx
-import pele_platform.Adaptive.solvent as sv
 import pele_platform.Adaptive.pca as pca
+import pele_platform.Adaptive.solvent as sv
+import pele_platform.Adaptive.ligand_parametrization as lg
```

Comment:
Maybe we can apply a small refactor and rename this module to plop_parametrization (to emphasize that it belongs to plop). In the future we will detect easily which part of the code we need to remove.